docs/references/apps/Connector.md

```diff
@@ -0,0 +1,156 @@
+# Connector
+
+The Connector app can automatically connect interfaces together. It does this using the query mechanisms inside HELICS to detect all the unconnected interfaces in a cosimulation. Then using given database rules it will establish connections between those interfaces.
```

Comment:
```suggestion
The Connector app can automatically connect interfaces together. It does this using the query mechanisms inside HELICS to detect all the unconnected interfaces in a cosimulation. Then using given configuration rules it will establish connections between those interfaces.
```